# 2022-Zoo-Biology-Husbandry-Report-Supplementary-Materials-Data-and-Code-
Contained in this directory are supplemental materials for Coleman et al. (2023) in Zoo Biology. 

The third-to-last sentence of the abstract of the paper should read, "Zoo researchers seeking microsatellite primer sets for their breeding toucan populations will likely benefit from our NOVEL markers, which can be leveraged both to assess relatedness and select breeding pairs."

The most important directory in this repository for readers of Coleman et al. (2023) is likely to be that named "KBTO Genotype", which lists  alleles (i.e., allele sizes) for the novel and heterologous loci in the study. This is especially useful given that Tables 2 and 3 in the publication do not list size ranges for the loci. There are duplicate files for all locus allele lists. 

In addition to all microsatellite data and analytical output (there is huge data overload in this repository——as you will be able to see all binning/scoring peaks from every PCR recipe/thermal cycler condition as I was optimizing my protocol——perhaps data sharing/availability is overrated...), some materials from a genomic/3RAD study I worked on for several years but have not published are available, including an in silico digest/size selection of the publicly available downy woodpecker genome. This genome was chosen to generate a a locus number estimate for the RADseq study, because woodpeckers are in the same order as toucans, which is appropriate (see a study out of the Manthey lab at Texas Tech [https://doi.org/10.1093/gbe/evy105], which suggests that Piciformes [woodpeckers, honeyguides, toucans, and barbets] all have uniquely high TE content in their genomes compared to other birds). The toucan species representative in Manthey et al. (2018) was the Green-billed Toucan (Ramphastos dicolorus). 

3RAD reads were cleaned on the STACKS pipeline created by Julian Catchen and Nicolas Rochette and are included in this repository, but unfortunately, raw reads cannot be located. 
